<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings</class>
 <widget class="QWidget" name="settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1281</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QFrame" name="frame_images_select">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="DescriptiveComboBox" name="image_getter_select" native="true"/>
         </item>
         <item>
          <widget class="FileDragDrop" name="photo_drop" native="true"/>
         </item>
         <item>
          <widget class="FileDragDrop" name="video_drop" native="true"/>
         </item>
         <item>
          <widget class="DescriptiveComboBox" name="cam_id_select" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_memorizer">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Memorizer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="DescriptiveComboBox" name="memorizer_select" native="true"/>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_lower_moves_first">
           <property name="text">
            <string>Lower moves  first</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_siren">
           <property name="styleSheet">
            <string notr="true">color: rgb(224, 27, 36);</string>
           </property>
           <property name="text">
            <string>Use siren (LOUD)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_config">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Config</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_ok">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="DetectorsSelect" name="corner_and_inventory_select" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DescriptiveComboBox</class>
   <extends>QWidget</extends>
   <header>GUI/widgets/DescriptiveComboBox.h</header>
   <container>1</container>
   <slots>
    <signal>element_changed(QVariant)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>DetectorsSelect</class>
   <extends>QWidget</extends>
   <header>GUI/widgets/DetectorsSelect.h</header>
   <container>1</container>
   <slots>
    <signal>splitter_changed(QVariant)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>FileDragDrop</class>
   <extends>QWidget</extends>
   <header>GUI/widgets/FileDragDrop.h</header>
   <container>1</container>
   <slots>
    <signal>received_content(QVariant)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>image_getter_select</sender>
   <signal>element_changed(QVariant)</signal>
   <receiver>settings</receiver>
   <slot>on_image_getter_changed(QVariant)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>photo_drop</sender>
   <signal>received_content(QVariant)</signal>
   <receiver>settings</receiver>
   <slot>on_photo_input(QVariant)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>video_drop</sender>
   <signal>received_content(QVariant)</signal>
   <receiver>settings</receiver>
   <slot>on_video_input(QVariant)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>memorizer_select</sender>
   <signal>element_changed(QVariant)</signal>
   <receiver>settings</receiver>
   <slot>on_memorizer_changed(QVariant)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>corner_and_inventory_select</sender>
   <signal>splitter_changed(QVariant)</signal>
   <receiver>settings</receiver>
   <slot>on_splitter_changed(QVariant)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_lower_moves_first</sender>
   <signal>clicked(bool)</signal>
   <receiver>settings</receiver>
   <slot>on_lower_moves_first_switched(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>-4</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cam_id_select</sender>
   <signal>element_changed(QVariant)</signal>
   <receiver>settings</receiver>
   <slot>on_cam_id_changed(QVariant)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_siren</sender>
   <signal>clicked(bool)</signal>
   <receiver>settings</receiver>
   <slot>on_alarm_switched(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>-2</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_ok</sender>
   <signal>clicked()</signal>
   <receiver>settings</receiver>
   <slot>on_ok_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>settings</receiver>
   <slot>on_cancel_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_image_getter_changed(QVariant)</slot>
  <slot>on_photo_input(QVariant)</slot>
  <slot>on_video_input(QVariant)</slot>
  <slot>on_memorizer_changed(QVariant)</slot>
  <slot>on_alarm_switched(bool)</slot>
  <slot>on_lower_moves_first_switched(bool)</slot>
  <slot>on_cam_id_changed(QVariant)</slot>
  <slot>on_splitter_changed(QVariant)</slot>
  <slot>on_ok_clicked()</slot>
  <slot>on_cancel_clicked()</slot>
 </slots>
</ui>
